Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2999,6 +3000,15 @@ extension BrowserViewController: IntroViewControllerDelegate {
             }
         }
     }
+    
+    func introViewControllerShouldAskForNotifications(_ introViewController: IntroViewController) -> Bool {
+        // Only include the slide if Leanplum has decided it needs to run and if the current locale is en_US/GB/CA/AU/HK/SG
+        return Leanplum.hasStarted() && Locale.current.identifier.startsWith("en_")
```

Comment:
Is there a chance Leanplum will not have started in time here? 

Starting Leanplum involves a network request which may not have returned in time.
```
This will initiate a call to Leanplum's servers to get the values
of the variables used in your app.
```